tests/Unit/Elliptic/DiscontinuousGalerkin/Test_ImposeBoundaryConditions.cpp

```diff
@@ -0,0 +1,487 @@
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{4., 5., 6.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{24., 25., 26.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{7., 8., 9.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{27., 28., 29.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{10., 11., 12.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{30., 31., 32.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{13., 14., 15.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{33., 34., 35.}}}}}}};
+
+  auto start_box = [
+    &mesh, &self_id, &west_id, &south_id, &coordmap, &data,
+    &external_directions, &external_mortar_ids
+  ]() noexcept {
+    const Element<2> element(self_id,
+                             {{Direction<2>::lower_xi(), {{west_id}, {}}},
+                              {Direction<2>::upper_eta(), {{south_id}, {}}}});
+
+    auto map = ElementMap<2, Frame::Inertial>(self_id, coordmap->get_clone());
+
+    db::item_type<bdry_vars_tag> bdry_vars;
+    for (const auto& direction : external_directions) {
+      auto& flux_vars = bdry_vars[direction];
+      flux_vars.initialize(3);
+      get<field_tag>(flux_vars) = data.bdry_vars.at(direction);
+    }
+
+    db::item_type<normal_dot_fluxes_tag<flux_comm_types>>
+        bdry_normal_dot_fluxes;
+    for (const auto& direction : external_directions) {
+      auto& flux_vars = bdry_normal_dot_fluxes[direction];
+      flux_vars.initialize(3);
+      get<Tags::NormalDotFlux<field_tag>>(flux_vars) =
+          data.bdry_fluxes.at(direction);
+    }
+
+    db::item_type<other_data_tag> bdry_other_data;
+    for (const auto& direction : external_directions) {
+      auto& other_data_vars = bdry_other_data[direction];
+      other_data_vars.initialize(3);
+      get<OtherData>(other_data_vars) = data.bdry_other_data.at(direction);
+    }
+
+    db::item_type<normal_dot_fluxes_tag<flux_comm_types>>
+        exterior_bdry_normal_dot_fluxes;
+    for (const auto& direction : external_directions) {
+      auto& flux_vars = exterior_bdry_normal_dot_fluxes[direction];
+      flux_vars.initialize(3);
+      get<Tags::NormalDotFlux<field_tag>>(flux_vars) =
+          data.exterior_bdry_fluxes.at(direction);
+    }
+
+    db::item_type<other_data_tag> exterior_bdry_other_data;
+    for (const auto& direction : external_directions) {
+      auto& other_data_vars = exterior_bdry_other_data[direction];
+      other_data_vars.initialize(3);
+      get<OtherData>(other_data_vars) =
+          data.exterior_bdry_other_data.at(direction);
+    }
+
+    db::item_type<exterior_bdry_vars_tag> exterior_bdry_vars;
+    for (const auto& direction : external_directions) {
+      auto& vars = exterior_bdry_vars[direction];
+      vars.initialize(3);
+      get<field_tag>(vars) = data.exterior_bdry_vars.at(direction);
+    }
+
+    int initial_temporal_id = 0;
```

Comment:
can these be const? should they be `size_t`s?

Reply:
I set up a temporal id with type `int` at the beginning of the file. It's only for testing, could be any time.